public/omni/troubleshooting/troubleshooting.mdx

```diff
@@ -0,0 +1,31 @@
+---
+title: "Troubleshooting"
+description: "Troubleshooting Omni and machines"
+---
+
+This guide will assume you already have Omni available and running in your environment either self-hosted or using the Omni SaaS.
+The sections are unordered and can be used to troubleshoot your specific situation or error messages.
+
+## Removing stuck machines
+
+Machines connected to Omni can get in states where Omni can no longer cleanly remove them. This may be because the machine is no longer available, the machine has errors, or other reasons.
+
+This may block the ability to delete a cluster as Omni tries to cleanly remove machines before deleting the clutser resoures. Sometimes trying to force delete the machine from the Omni UI also does not remove the machine.
```

Comment:
Force delete from the UI goes exactly what `omnictl delete link` does, fyi.

I would document two paths, but there is no point in doing both.

Reply:
nvm, I found the pop-up after a machine was in a stuck state

<img width="616" height="292" alt="image" src="https://github.com/user-attachments/assets/705a5d75-258c-4df9-bc95-753be27a5870" />